src/packages/logger/request-logger/templates.js

```diff
@@ -77,3 +79,20 @@ Processed ${cyan(`${method}`)} "${path}" ${magenta('Params')} ${
   : null
 }
 `;
+
+/**
+ * @private
+ */
+const pad = (endTime, startTime, duration) => {
+  const maxLength = (endTime - startTime).toString().length;
+  let padNum = maxLength - duration.toString().length;
+
+  let padding = '';
+
+  while (padNum > 0) {
+    padding += ' ';
+    padNum--;
+  }
```

Comment:
This function can be simplified to determine the amount of padding needed with `Math` functions rather than counting the string length. [Inspired by this post on s/o](http://stackoverflow.com/questions/14879691/get-number-of-digits-with-javascript).

```javascript
function countDigits(num: number) {
  return Math.floor(Math.log10(num) + 1);
}

function pad(startTime: number, endTime: number, duration: number) {
  const maxLength = countDigits(endTime - startTime);

  return ' '.repeat(maxLength - countDigits(duration)) + duration;
}
```